testdata/baselines/reference/fourslash/documentSymbols/navbar01.baseline.md

```diff
@@ -0,0 +1,602 @@
+// === Document Symbols ===
+```ts
+// @FileName: /navbar01.ts
+// Interface
+interface IPoint {
+    getDist(): number;
+    new(): IPoint;
+    (): any;
+    [x:string]: number;
+    prop: string;
+}
+
+/// Module
+module Shapes {
+    // Class
+    export class Point implements IPoint {
+        constructor (public x: number, public y: number) { }
+
+        // Instance member
+        getDist() { return Math.sqrt(this.x * this.x + this.y * this.y); }
+
+        // Getter
+        get value(): number { return 0; }
+
+        // Setter
+        set value(newValue: number) { return; }
+
+        // Static member
+        static origin = new Point(0, 0);
+
+        // Static method
+        private static getOrigin() { return Point.origin;}
+    }
+
+    enum Values { value1, value2, value3 }
+}
+
+// Local variables
+var p: IPoint = new Shapes.Point(3, 4);
+var dist = p.getDist();
+```
+
+# Symbols
+
+```json
+[
+	{
+		"name": "IPoint",
+		"kind": "Interface",
+		"range": {
+			"start": {
+				"line": 1,
+				"character": 0
+			},
+			"end": {
+				"line": 7,
+				"character": 1
+			}
```

Comment:
In some way, I wonder if we should also override printing of ranges in the baseline into something more compact.

Reply:
Crazy idea but what about printing inline like

```
/*Method getDist L2C4-L2C22*/«‹getDist›(): number»
```